srml/assets/src/lib.rs

```diff
@@ -82,10 +82,8 @@ decl_module! {
 	}
 }
 
-/// An event in this module. Events are simple means of reporting specific conditions and
-/// circumstances that have happened that users, Dapps and/or chain explorers would find
-/// interesting and otherwise difficult to detect.
 decl_event!(
+	/// An event in this module.
```

Comment:
Let's agree on one correct comment to go here, before the `pub enum Event` and use it for each module. I think "`/// Events for this module.`" would be ok.

Reply:
ok,
(as a note: if no module need to have a more explicit description, we can make it generated by the macro and remove the ability to customize)

Reply:
That is probably better, because the notion of "events" could change, and we don't want to update the same comment in the entire SRML. There is documentation (at least in progress) on the `Event` type. Perhaps it makes sense to have a more thorough comment in `srml/example/src/lib.rs` and `node-template/runtime/src/template.rs`. Everything else should be consistent unless there is a unique feature to a module.

Reply:
+1 for macro generation

Reply:
macro generation now as such:
```
/// Events for this module
///
/// (other doc of user if some)
```